packages/image_picker/image_picker_platform_interface/lib/src/types/media_selection_options.dart

```diff
@@ -0,0 +1,48 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:image_picker_platform_interface/image_picker_platform_interface.dart';
+
+/// Options class for [ImagePickerPlatform.getMedia].
+class MediaSelectionOptions {
+  /// Construct a new MediaSelectionOptions instance.
+  MediaSelectionOptions({
+    List<RetrieveType>? types,
+    this.maxImageWidth,
+    this.maxImageHeight,
+    this.imageQuality,
+    this.allowMultiple = false,
```

Comment:
Do we need `maxDuration` here for videos? Or better, a `VideoSelectionOptions`?

More generally, it seems like we need a plan around how to handle type-specific options. Should this method always take the union of all options for the other methods?